code/game/gamemodes/wizard/wizard.dm

```diff
@@ -19,44 +19,46 @@
 
 	var/finished = 0
 
+	var/const/players_per_wizard = 10
 	var/const/waittime_l = 600 //lower bound on time before intercept arrives (in tenths of seconds)
 	var/const/waittime_h = 1800 //upper bound on time before intercept arrives (in tenths of seconds)
+	var/wizard_amount= 1
 
 	can_be_mixed = TRUE
 
 /datum/game_mode/wizard/announce()
 	to_chat(world, "<B>The current game mode is - Wizard!</B>")
-	to_chat(world, "<B>There is a <span class='danger'>SPACE WIZARD on the station. You can't let him achieve his objective!</span>")
+	to_chat(world, "<B>There are a number of <span class='danger'>SPACE WIZARDS on the station. You can't let them achieve their objective!</span>")
 
 /datum/game_mode/wizard/pre_setup()
 	var/list/datum/mind/possible_wizards = get_players_for_role(ROLE_WIZARD)
 	if(possible_wizards.len==0)
 		log_admin("Failed to set-up a round of wizard. Couldn't find any volunteers to be wizards.")
 		message_admins("Failed to set-up a round of wizard. Couldn't find any volunteers to be wizards.")
 		return 0
-	var/datum/mind/wizard
-	while(possible_wizards.len)
-		wizard = pick(possible_wizards)
+
+	wizard_amount= max(round(num_players()/players_per_wizard, 1), 1)
+
+	for(var/j = 0, j < wizard_amount, j++)
+		if(!possible_wizards.len)
+			break
+		var/datum/mind/wizard = pick(possible_wizards)
+		possible_wizards -= wizard
 		if(wizard.special_role || (mixed && (wizard in ticker.mode.modePlayer)))
-			possible_wizards -= wizard
-			wizard = null
 			continue
```

Comment:
If you don't want the number of wizards to be reduced if this block executes, you can throw a `j--` above this